src/pages/DashboardHome.vue

```diff
@@ -55,15 +55,19 @@
                         </tr>
                     </tbody>
                 </table>
-
-                <div class="d-flex justify-content-center kuma_pagination">
+                <div v-if="!showAll" class="d-flex justify-content-center kuma_pagination">
                     <pagination
                         v-model="page"
                         :records="importantHeartBeatList.length"
                         :per-page="perPage"
                         :options="paginationConfig"
                     />
                 </div>
+                <div v-if="importantHeartBeatList.length > perPage" class="d-flex justify-content-center my-1">
+                    <button class="btn btn-link" @click="toggleShowAllRecords">
+                        {{ showAll ? $t("Show Less") : $t("Show All") }}
```

Comment:
In general:
please add additional translation keys to the `en.json` (and any other language for which you are a native speaker) file.
If they are not available in this file, they won't be [available for translation on weblade](https://github.com/louislam/uptime-kuma/blob/master/src/lang/README.md).